lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -58,6 +58,18 @@ function getDefinitionsToRun(allTestDefns, requestedIds, {invertMatch}) {
     console.log(usage);
   }
 
+  if (onlyUrls) {
```

Comment:
interaction with `invertMatch` is a little confusing now, maybe warrants some documentation on the method and/or CLI help?

long-term fix might be a `skipUrls` parallel :)